### ISSUE TITLE
Export Error in Umbraco 17.3.3

### ISSUE THREAD

**Issue by Netcodr81** [mattbegent/umbraco-accessibility-reporter#48]
Issue: I am unable to export my accessibility results via the back office. When looking at the console in developer tools I see the following:

[

<img width="1186" height="274" alt="Image" src="https://github.com/user-attachments/assets/e168d78c-5295-49ca-aa3b-8e35f70058a2" />

](url)

Details:
Umbraco.CMS - 17.3.3
Chrome - 147.07727.101
Edge - 147.0.3912.60
Umbraco.Community.AccessibilityReporter - 4.2.0

**Comment by mattbegent** [comment]
Hi @Netcodr81, thanks for reporting this. I can replicate the issue in  Umbraco 17.3.3. I'll have a look at fixing it.

**Comment by mattbegent** [comment]
Hi @Netcodr81, I found the issue. Umbraco routing changes were intercepting download behaviour. This should be fixed in 4.2.1. Let me know if this fix doesn't work for you. Thanks! https://www.nuget.org/packages/Umbraco.Community.AccessibilityReporter/4.2.1
